src/test/scala/org/broadinstitute/dsde/agora/server/webservice/ApiServiceSpec.scala

```diff
@@ -73,6 +77,7 @@ with AgoraTestData with AgoraDbTest {
       assert(agoraEntity.payload === payload1)
       assert(agoraEntity.snapshotId !== None)
       assert(agoraEntity.createDate != null)
+      assert(status === Created)
```

Comment:
I would add the phrase "with a status 201" to the test description so that it is explicit.  For instance "Agora" should "create and return a method with a status of 201"